### ISSUE TITLE
Wrong URI composition when ServicerPort is a reference to a label

### ISSUE THREAD

**Issue by lzecca78** [stakater/Forecastle#56]
I have several ingress configured as following : 
``` apiVersion: extensions/v1beta1
  kind: Ingress
  metadata:
    name: alertmanager
    namespace: monitoring
    annotations:
      forecastle.stakater.com/expose: 'true'
      forecastle.stakater.com/icon: 'https://github.com/stakater/ForecastleIcons/raw/master/prometheus.png'
      kubernetes.io/ingress.class: internal
      nginx.ingress.kubernetes.io/force-ssl-redirect: 'true'
    labels:
      app: alertmanager
  spec:
    rules:
      - host: alertmanager.examples.com
        http:
          paths:
            - path: '/'
              backend:
                serviceName: alertmanager-main
                servicePort: web
```

Where `servicePort` is the label defined in the service. 

### what is happening ###
Forecastle is composing the uri like `http://alertmanager.examples.comweb/`. 

### how it should behave ###
it should ignore the `servicePort` param because the ingress expose them in 80/443 and compose the uri like `http://alertmanager.examples.com`

**Comment by lzecca78** [comment]
is there any news about this?

**Comment by kahootali** [comment]
We just released v1.0.18 that fixes this. Closing this issue as of now, please re-open if you face the issue.

**Comment by lzecca78** [comment]
it works perfectly now! Really thanks!
